app/src/lib/stores/app-store.ts

```diff
@@ -4677,7 +4753,145 @@ export class AppStore extends TypedBaseStore<IAppState> {
 
     this.emitUpdate()
   }
+
+  /** This shouldn't be called directly. See `Dispatcher`. */
+  public async _createStash(
+    repository: Repository,
+    branchName: string,
+    removePreviousStash: boolean = true
+  ) {
+    const { branchesState } = this.repositoryStateCache.get(repository)
+    const { tip } = branchesState
+    if (tip.kind !== TipState.Valid) {
+      return
+    }
+
+    // get the previous stash before we create a new one
+    const previousStash = removePreviousStash
+      ? await getLastDesktopStashEntryForBranch(repository, branchName)
+      : null
+
+    await createDesktopStashEntry(repository, branchName)
+
+    if (previousStash === null) {
+      return
+    }
+
+    await dropDesktopStashEntry(repository, previousStash.stashSha)
+    log.warn(
```

Comment:
This could be `log.info`

Reply:
Why was this marked as resolved? Tentatively marking as unresolved.

Reply:
fixed in https://github.com/desktop/desktop/pull/7210/commits/0ef307e095ea0d1633dc9af2da6003bb46f2e202

Reply:
Resolved in https://github.com/desktop/desktop/pull/7210/commits/0ef307e095ea0d1633dc9af2da6003bb46f2e202